lib/k8s/operation.ex

```diff
@@ -173,31 +189,48 @@ defmodule K8s.Operation do
       end
 
     query_params =
-      case verb do
-        :apply ->
+      cond do
```

Comment:
Just wanted to callout that when I fixed the tests, I had to adjust this due to the verb change from apply -> patch